toDoList/src/CRUD.js

```diff
@@ -0,0 +1,45 @@
+import ToDoList from './toDoList';
+
+const toDoList = new ToDoList();
+
+const newTask = (description) => {
+  toDoList.toDoTasks = JSON.parse(localStorage.getItem('toDoList'));
+  const completedDefault = false;
+  const index = toDoList.toDoTasks.length + 1;
```

Comment:
- [ ] **[OPTIONAL]** It is better to use let instead of const for the numeric values.